testdata/serviceconfig.yml

```diff
@@ -7,6 +7,23 @@ rules:
   aspects:
   - kind: quotas
     params:
+  - kind: metrics
+    adapter: "prometheus"
+    params:
+      metrics:
+      - descriptor: "request_count"
+        value: "1"
```

Comment:
The `value` field is a string expression, it just so happens that the expression I want evaluated is the literal `1`. I can make it look like `1` since yaml doesn't require quotes for strings, but in reality it's still a string expression that is evaluated to the integer 1.

I agree it's odd, but it's what we want to do: for each (request, service, target, response code) set we want to increment the counter by 1. I'll add comments explaining.